.github/workflows/update-search.yml

```
@@ -0,0 +1,44 @@
+  schedule:
+    - cron: '0 4 * * *'  # Run daily at 4:00 UTC
+  workflow_dispatch:  # Allow manual trigger
+
+# Add permissions needed for the workflow
+permissions:
+  contents: write    # This allows the action to commit and push changes
+
+jobs:
+  update-search:
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)